tutorials/fundamentals/concat-regexp.md

```diff
@@ -0,0 +1,14 @@
+To concatenate a regular expression in JavaScript, you can use a combination
+of the `+` operator, and the `new RegExp()` class:
+
+```javascript
+let reg = /mastering/g;
+let exp = /js/y;
+let flags = reg.flags+exp.flags;
```

Comment:
Let's put this logic (L7-L9) as a separate `function concatRegexp()` function.

Also, space before and after `+` please